torch/csrc/distributed/rpc/init.cpp

```diff
@@ -164,6 +164,19 @@ Otherwise, throws an exception.
     agent->start();
   });
 
+  module.def(
+      "shutdown",
+      []() {
+        auto agent = RpcAgent::getDefaultRpcAgent();
+        agent->shutdown();
+      },
+      R"(
+      Locally shuts down the running RPC agent, without sending messages to
```

Comment:
I am not sure if this paragraph should start without indent. Please double check if the doc format is expected

Reply:
<img width="910" alt="pybind" src="https://user-images.githubusercontent.com/8039770/69399377-c3f3ee00-0ca2-11ea-85e1-9b926d2979b7.png">

This is how the doc looks, indentation looks fine